### DISCUSSION TITLE
Checkbox checked for option(s) and group

### DISCUSSION THREAD

**Discussion by photomakes** [shentao/vue-multiselect#1728] [Q&A]
Hello. Help me correctly set the checkbox for an option, if the option is selected and if the options are selected by group label.

	<div id="groupSelect">
		<multiselect
			v-model="value"
			:options="options"
			placeholder="..."
			label="name"
			track-by="name"
			:multiple="true"
			:close-on-select="false"
			group-values="groupList"
			group-label="groupName"
			:group-select="true"
			:show-labels="false"
			:disabled="isDisabled"
			@select="onSelect($event)"
			@remove="onRemove($event)">
			<template slot="selection" slot-scope="{ values, search, isOpen }">
				<span class="multiselect__single" v-if="values.length" v-show="!isOpen">
					Selected: {{ values.length }}
				</span>
			</template>
	
			<template slot="option" slot-scope="props">
				<div v-if="props.option.$isLabel">
					<input type="checkbox" v-model="props.option.checked">
	
					{{ props.option.$groupLabel }}
				</div>
	
				<div v-else>
					<input type="checkbox" v-model="props.option.checked">
	
					{{ props.option.name }}
				</div>
			</template>
		</multiselect>
	</div>
	
	<script type="module">
		var app = new Vue({
			el: '#groupSelect',
			components: { Multiselect: window.VueMultiselect.default },
			data () {
				return {
					isDisabled: false,
					value: [],
					options: [
						{
							groupName: 'Select all',
							groupList: [
								{
									name: 'Option name',
									checked: false,
								},
								{
									name: 'Option name',
									checked: false,
								},
								{
									name: 'Option name',
									checked: false,
								},
							]
						}
					]
				}
			},
			methods: {
				onSelect (option) {
					option.checked = true;
				},
				onRemove (option) {
					option.checked = false;
				}
			}
		})
	</script>

**Comment by photomakes** [comment]
The popularity of the project is not justified.

**Comment by mattelen** [comment]
I'm not quite understanding what you are trying to achieve here. If you are still interested, feel free to pop your code in a JSfiddle - this makes it easier to run and get help
